src/python/pants/testutil/rule_runner.py

```diff
@@ -126,7 +126,9 @@ def __init__(
             execution_options=ExecutionOptions.from_bootstrap_options(global_options),
         ).new_session(
             build_id="buildid_for_test",
-            session_values=SessionValues({OptionsBootstrapper: options_bootstrapper}),
+            session_values=SessionValues(
+                {OptionsBootstrapper: options_bootstrapper, PantsEnvironment: PantsEnvironment()}
```

Comment:
The default `PantsEnvironment` is `os.environ`. It should probably have required args like `OB.create` does... but if not, we should make sure that it is empty here.

Reply:
Fixed. It defaults to being empty now.